### DISCUSSION TITLE
How to properly position assets on a hud?

### DISCUSSION THREAD

**Discussion by crapthings** [rexrainbow/phaser3-rex-notes#242] [Q&A]
![image](https://user-images.githubusercontent.com/1147704/153553093-bd73f1ad-9b9f-4fe1-9524-e25463017f43.png)

should I use it as a whole or a separated layer?

how do I position images like avatar, hp bar on it and it should fit with all mobile devices?

**Comment by rexrainbow** [comment]
HUD might
- Set `scrollFactor` to `0`, to against camera scrolling
- Put on a Layer object, to against z-order of game objects
- Put on a separated scene, to against camera zooming
